pkg/bindinfo/binding_cache.go

```diff
@@ -86,6 +105,31 @@ func (fbc *fuzzyBindingCache) FuzzyMatchingBinding(sctx sessionctx.Context, fuzz
 					break
 				}
 			}
+		} else {
+			missingSQLDigest = append(missingSQLDigest, sqlDigest)
+		}
+	}
+	return matchedBinding, leastWildcards, isMatched, missingSQLDigest
+}
+
+func (fbc *fuzzyBindingCache) slowGet(sctx sessionctx.Context, missingfuzzyDigest []string, tableNames []*ast.TableName, enableFuzzyBinding bool, leastWildcards int) (matchedBinding Binding, isMatched bool) {
+	for _, sqlDigest := range missingfuzzyDigest {
```

Comment:
`fuzzyDigest` and `sqlDigest` are two different terms. `sqlDigest := fuzzyDigest` seems incorrect, please be aware of which digest you want to use here.

Reply:
it has been fixed. It is a wrong name.